emperor/support_files/js/emperor.js

```diff
@@ -2257,15 +2271,15 @@ $(document).ready(function() {
 
 		// build divs to hold point labels and position them
 		var labelshtml = "";
-		for(var i in g_plotIds) {
-			var sid = g_plotIds[i];
+	        _.each(g_plotIds, function(pltID){
```

Comment:
Why not use _.each(g_plotIds, function(sid){ and remove the next line. This also applies to the other instances.